templates/sag-db-sqlserver/README.md

```diff
@@ -18,8 +18,7 @@ With this template you can create user, database, and webMethods database schema
 
 ## Running as a standalone Composite Template
 
-Run this command to install Database Component Configurator on `local` 10.2 managed node and create
-webMethods `webm` database, `webm` user and all schemas for 10.1 release on `sqlserver` SQL server connecting with `sa` user and `MaNaGe123` password:
+To install Database Component Configurator on a node with alias `local` and version 10.2 and create a webMethods database, named `webm` with credentials `webm`/`webm` and all database schemas for version 10.2 on a Microsoft SQL Server host connecting with `sa` /`MaNaGe123`: 
```

Comment:
To install Database Component Configurator 10.1 on the Command Central node with alias `local`, create a database named `webm` and a database user named `webm` with password `webm`, and create database components for 10.1 Integration Server and My webMethods Server products using user `sa` with password `MaNaGe123`: